### PR TITLE
Added cards for different guides in Introduction page

### PR DESCRIPTION
**Pull request by AshwinSriram11**
According to the issue #82 , I have added links to different guides on the Introduction page and removed the unwanted content (Get-Started section) from the Introduction page .

This PR solves a part of #82 .

> Small boxes that redirect people to what they care about, similar to https://nextjs.org/docs#join-our-community

### CODE REVIEW

File: content/docs/getting-started/introduction.md [hunk 1]
```diff
@@ -22,32 +22,37 @@ The Prometheus operator includes, but is not limited to, the following features:
...
+  title="API Reference"
+  href="https://prometheus-operator.dev/docs/api-reference/api/"
+  description="Reference for different fields of Custom Resources in Prometheus-Operator."
+>}}
+
+<!-- Developer Guide -->
+{{<link-card
+  title="Developer Guide"
+  href="https://prometheus-operator.dev/docs/developer/getting-started/"
+  description="Learn how to configure monitoring for your applications."
```
Comment (ArthurSens) [inline content/docs/getting-started/introduction.md:43]:
```suggestion
  description="Learn how to configure scraping, alerting, and recording rules for your applications."
```

File: content/docs/getting-started/introduction.md [hunk 2]
```diff
@@ -22,32 +22,37 @@ The Prometheus operator includes, but is not limited to, the following features:
...
+>}}
+
+<!-- Platform Guide -->
+{{<link-card
+  title="Platform Guide"
+  href="https://prometheus-operator.dev/docs/platform/webhook/"
+  description="Set up, configure and manage instances of Prometheus and related resources."
```
Comment (ArthurSens) [inline content/docs/getting-started/introduction.md:50]:
```suggestion
  description="Set up, configure and manage instances of Prometheus-Operator, Prometheus, Alertmanager and ThanosRuler resources."
```

**Review by ArthurSens — COMMENTED** [review]
This looks so cool! 😍 

I have just some super small recommendations about the wording, but I'm also aware that the text might be too long for the card.

What do you think?

**Comment by AshwinSriram11** [comment]
>I have just some super small recommendations about the wording, but I'm also aware that the text might be too long for the card.

>What do you think?

I was trying to be mindful of that and went with one-liners. But we can add the changes if we make every card with same lines. Then it would not look weird.

**Comment by AshwinSriram11** [comment]
> I was trying to be mindful of that and went with one-liners. But we can add the changes if we make every card with same number of lines. Then it would not look weird.

I tried it locally and I think it looks good. Let me make a commit.

**Review by ArthurSens — APPROVED** [review]
LGTM!!

File: layouts/shortcodes/card-grid.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="card-nav d-flex flex-column flex-sm-row" gap="20">
+    {{ .Inner }}
+  </div>
```
Comment (nicolastakashi) [inline layouts/shortcodes/card-grid.html:3]: nit you have an extra space on this line

**Review by nicolastakashi — APPROVED** [review]
🚀

**Pull request merged**